src/features/ar.ts

```diff
@@ -17,7 +17,7 @@ import {property} from 'lit-element';
 
 import {IS_ANDROID, IS_AR_QUICKLOOK_CANDIDATE, IS_IOS, IS_WEBXR_AR_CANDIDATE} from '../constants.js';
 import ModelViewerElementBase, {$container, $renderer, $scene} from '../model-viewer-base.js';
-import {Constructor, deserializeUrl} from '../utils.js';
+import {Constructor, deserializeUrl} from '../utilities';
```

Comment:
Previously I asked for these to be converted to include their file extension

Reply:
TS fails to build with the extension.

Example failure message:
```
src/features/controls.ts:23:27 - error TS2691: An import path cannot end with a '.ts' extension. Consider importing '../utilities' instead.
```

Reply:
You need to refer to paths with the `.js` extension

Reply:
My bad. Fixed.

Reply:
\o/ let me give this puppy a spin...